packages/profiling-browser/test/integration.test.ts

```diff
@@ -0,0 +1,97 @@
+import type { Event } from '@sentry/types';
+import { JSDOM } from 'jsdom';
+
+import { getCurrentHub } from '../../core/src/hub';
+import { BrowserProfilingIntegration, PROFILING_EVENT_CACHE, sendProfile } from '../src/integration';
+
+describe('BrowserProfilingIntegration', () => {
+  beforeEach(() => {
+    // Clear profiling event cache
+    PROFILING_EVENT_CACHE.clear();
+
+    const dom = new JSDOM();
+    // @ts-ignore need to override global document
+    global.document = dom.window.document;
+    // @ts-ignore need to override global document
+    global.window = dom.window;
+    // @ts-ignore need to override global document
+    global.location = dom.window.location;
```

Comment:
m: We should add an `afterAll` or `afterEach` to reset these global values to their previous values to avoid test bleeding.

Reply:
Oh yeah, you are right. I just copied this from tracing - looks like the resets are missing there too